ops/prod/app_gateway_url_redirects.tf

```diff
@@ -184,6 +184,7 @@ resource "azurerm_application_gateway" "www_redirect" {
     rule_type                   = "Basic"
     http_listener_name          = "${local.name}-http"
     redirect_configuration_name = "httpsRedirect"
+    rewrite_rule_set_name       = "HSTS_ReWrite"
```

Comment:
I don't remember seeing the `www_redirect` resource being at issue, but I also have no problem implementing the rule set here if you or CSPO think it's necessary. So long as this doesn't cause issues, I'm totally okay with it!